apps/bfd-server/dev/api-changelog.md

```diff
@@ -1,5 +1,11 @@
 # API Changelog
 
+## BFD-1461 Implement Handling for Null Enrollment Reference Years
+
+ * Following FHIR mapping changes were made:
+
+  * For V1 and V2, if there is a reference year, return data normally.  For V1 and V2, if there is a NULL reference year, do NOT return data that is associated with that specific reference year. All other data is returned as expected 
+
```

Comment:
Can we make this a bit more explanatory as to what data will and will not be returned when the reference year is null? @bfausett77 any recommendations on what we put here for this one?

Reply:
I think that would be good.  I am not sure what the associated fields, to the reference year, are though.  @cbrunefearless may know better than I.

Reply:
I referenced them in the json, or did u want me to reference them as fields in a comma list?

Reply:
what you have looks good to me